### DISCUSSION TITLE
Pseudo spectra extraction

### DISCUSSION THREAD

**Discussion by NotValdemaras** [vdemichev/DiaNN#385] [Q&A] [answered]
Hi, 

Is it possible to extract precursor-fragment pseudo spectra like with DIA-Umpire, with DIA-NN?

**Comment by vdemichev** [comment, marked as answer]
Yes, please use the experimental --mgf command. The performance of this is NOT optimised and will be significantly improved in the future. Modern technologies like Scanning SWATH and dia-PASEF are poorly supported by this feature.

**Reply by NotValdemaras** [reply]
Great! Thank you. I will test it out.

**Reply by heylookabed** [reply]
Hello, thank you for the community support with DIA-NN.  We have tried using the --mgf command with 1.8.1, but no mater how we attempt to implement, it results in 0 data output. Is there a trick to using this command or is there yet another way to generate a peak-picked / deconvoluted output file (e.g., mzML)?

**Reply by vdemichev** [reply]
I could take a look at the logs
